docs/src/toml.md

```diff
@@ -35,6 +36,19 @@ type = "float"
 description = "Molecular weight dry air (kg/mol)"
 ```
 
+### Properly tagged parameter
+The parameter below has tags of varying specificity, from the package name to the specific parameter set it is intended for.
+
+```TOML
+[prandtl_number_0_grachev]
```

Comment:
Highlight that tags are optional (this is not a namespace), and so the name must still be globally unique